invenio/templates/configuration.yaml

```diff
@@ -38,7 +38,7 @@ metadata:
 data:
   invenio.conf: |
     upstream web {
-        server web:5000 max_conns={{ .Values.nginx.max_conns }};
```

Comment:
It could but I found it more complex since the behavior then would be Web Service(Load Balancer) -> POD -> NGINX -> Web Service(maybe now different POD) -> Web application
Instead, now it's  Web Service(Load Balancer) -> POD -> NGINX -> Web application.

Reply:
Totally true, makes sense. Thanks!